clang/docs/ReleaseNotes.rst

```diff
@@ -171,6 +171,9 @@ C++23 Feature Support
 ^^^^^^^^^^^^^^^^^^^^^
 - Removed the restriction to literal types in constexpr functions in C++23 mode.
 
+- Extend lifetime of temporaries in mem-default-init for P2718R0. Clang now fully
+  supported `P2718R0 Lifetime extension in range-based for loops <https://wg21.link/P2718R0>`_.
```

Comment:
Typo: "Clang now fully supports ...", not "supported"

Reply:
Fixed.